devel/meson/files/patch-meson-gnome.diff

```diff
@@ -1,24 +1,6 @@
-Due to a presumed bug in MP's g-ir-scanner[1], generated typelib files often
-point to the dylib's relative build path. E.g. running
-
-    g-ir-inspect --print-shlibs Pango
-
-Will print something like
-
-    shlib: ./pango/libpango-1.0.0.dylib
-
-At run-time, the GObject Introspection infrastructure won't be able to find the
-libpango dylib, and panic ensues. This patch ensures that the full install
-paths are specified in the typelib file. You can ensure correct operation with
-the above command, which should print something like
-
-    shlib: /opt/local/lib/libpango-1.0.0.dylib
-
-[1] https://trac.macports.org/ticket/62391
-
```

Comment:
Why are you deleting this note?

Reply:
@evanmiller Thanks for pointing out. Probably I had a reason for that, but atm I have no idea why it was removed. Restoring it now.